### PR TITLE
:sparkles: run ci via bundle

### PR DESCRIPTION
**Pull request by djzager**

### CODE REVIEW

File: .github/workflows/global-ci-bundle.yml [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+          Namespace for the konveyor install.
+        type: string
+        required: false
+        default: ""
+      tackle_cr:
+        description: |
+          Full yaml encoded string representing the Tackle resource to be created.
+        type: string
+        required: false
+        default: ""
```
Comment (djzager) [inline .github/workflows/global-ci-bundle.yml:31]: This likely needs to be json encoded to make it easier to manage as input or from workflow dispatch (that can't handle multiline).

Comment (djzager) [reply .github/workflows/global-ci-bundle.yml:31]: I also wonder if there are certain things in the Tackle CR that MUST BE set in order to work correctly.

**Comment by djzager** [comment]
This needs https://github.com/konveyor/operator/pull/278

**Comment by djzager** [comment]
Verified the changes work in https://github.com/djzager/ci/actions/runs/6786233781/job/18446292708

**Comment by djzager** [comment]
Now also relies on https://github.com/konveyor/operator/pull/279

**Review by aufi — APPROVED** [review]
Fabian might have a better insight, but LGTM.

**Review by fabianvf — APPROVED** [review]
@djzager would it be possible to update the nightlies/main branch CI to use these? 
https://github.com/konveyor/ci/blob/main/.github/workflows/nightly-main.yaml

**Pull request merged**